e2e/playwright.config.js

```diff
@@ -36,8 +37,8 @@ export default defineConfig({
 
   projects: [
     {
-      name: 'logged-out',
-      testMatch: [/pages/],
+      name: 'tests',
+      testMatch: [/tests/],
```

Comment:
Could test files be where the components are? Or does mean the resulting images also have to be there?

It would be clearer to have the test files where Jest files are - with the code.

Reply:
Didn't yet look into that, good point.

Reply:
This might be worth a ticket of it's own since it might need some thinking and restructuring if possinble.